operator/internal/manifests/internal/config/loki-config.yaml

```
@@ -14,6 +14,7 @@ common:
       container_name: {{ .Container }}
       account_name: {{ .AccountName }}
       account_key: {{ .AccountKey }}
+      endpoint_suffix: {{ .EndpointSuffix }}
```

Should we make this optional in the output as well?

```suggestion
      {{- with .EndpointSuffix }}
      endpoint_suffix: {{ . }}
      {{- end }}
```